sklearn/decomposition/tests/test_online_lda.py

```diff
@@ -184,7 +184,8 @@ def test_lda_no_component_error():
     rng = np.random.RandomState(0)
     X = rng.randint(4, size=(20, 10))
     lda = LatentDirichletAllocation()
-    regex = r"^no 'components_' attribute"
+    regex = "This LatentDirichletAllocation instance is not fitted yet. " + \
+        "Call 'fit' with appropriate arguments before using this method."
     assert_raises_regexp(NotFittedError, regex, lda.transform, X)
```

Comment:
This already tested by the common test:

```
sklearn/tests/test_common.py::test_non_meta_estimators[LatentDirichletAllocation-LatentDirichletAllocation-check_transformers_unfitted]
```

We can remove it then

Reply:
I am afraid I am not sure where exactly this is tested. Could you please provide a permalink from the code so that I can see it more clearly?

Reply:
This is `estimator_check.py` This is the common test that you changed below. It is run for most of the estimator and the line that I paste is the relevant run for this test for LDA.